v1/s012-pyglet/pong.html

```diff
@@ -0,0 +1,616 @@
+                        jako svůj první titul v roce 1972 a odstartovala tak boom herního průmyslu.
+                    </div>
+                </section>
+                <section class="col-lg-12">
+                    <h2>Konstanty a stav hry</h2>
+                    <div>
+                        Hra Pong má jednoduchá pravidla. Musíme je ale umět
+                        vyjádřit v Pythonu a to není úplně jednoduché. 
+                        Pojdmě si srovnat v hlavě, co ve hře máme.
+                        <ol>
+                            <li>Hrací pole ve tvaru obdélníku s půlící čárou.</li>
+                            <li>Míček létající určitou rychlostí po hracím poli.</li>
+                            <li>Dvě pálky pohybující se vertikálně na krajích pole.</li>
+                            <li>Dvě počítadla skóre.</li>
+                        </ol>
+                        Hra bude pro 2 hráče, nebudeme programovat chování počítače.
+                        Každý z hráčů může ovládat svou pálku stiskem kláves.
+                        Jeden hráč šipkou nahoru a šipkou dolů a druhý hráč klávesami
+                        W a S.
+
+                        Stav hry dokážeme v Pythonu vyjádřit pomocí proměnných
+                        a konstant. To dává smysl, protože některé
+                        věci se ve hře mění (poloha pálek, poloha míčku, rychlost míčku, skóre)
+                        a některé ne (velikost hrací plochy, velikost pálek a
+                        míčku, pálek a míčku, poloha a velikost počítadel skóre). 
+                        Ze složitějších datových struktur použijeme seznam
+                        (list, ten už známe) a množinu (set), která je velmi podobná
+                        matematické množině. Zjednodušeně je to seznam, který se
+                        nestará o pořadí a stejné prvky v něm mohou být právě jednou.
+                    </div>
+
+                    <div>
+                        Založte si nový skript a zkuste v Pythonu nadefinovat
+                        to jednodušší - <strong>konstanty</strong>, které 
+                        budeme v tvorbě hry potřebovat. Jen připomeňme, že
+                        názvy konstant se v Pythonu píší velkými písmeny.
+<details class="solution notes"><h4>Řešení</h4>
+<pre>
+# Velikost okna (v pixelech)
+SIRKA = 900
+VYSKA = 600
+
+VELIKOST_MICE = 20
+TLOUSTKA_PALKY = 10
+DELKA_PALKY = 100
+RYCHLOST = 200  # v pixelech za sekundu
+RYCHLOST_PALKY = RYCHLOST * 1.5  # taky v pixelech za sekundu
+
+DELKA_PULICI_CARKY = 20
+VELIKOST_FONTU = 42
+ODSAZENI_TEXTU = 30
+</pre>
+</details>
+
+                    </div>
+                    <div>
+                        Zkuste se znovu zamyslet a nadefinovat <strong>proměnné</strong>
+                        potřebné v naší hře jako je poloha míčku, poloha pálek,
+                        rychlost míče, stisknuté klávesy nebo skóre.
```

Comment:
Tady bych nedal "jako je", ale přímo seznam -- ".. v naší hře: polohu míčku ..."
Ani já bych nedokázal myslet tak dopředu, abych nadefinoval všechny proměnné. Nejlepší by bylo konstanty/proměnné připisovat postupně, jak budou potřeba během vznikání programu, ale to už by asi bylo moc veliké překopání. Takže bych tu asi řekl (napsal), že většinou se konstanty/proměnné -- stejně jako importy -- přidávají v průběhu, ale pro zjednodušení si napíšeme všechny už teď.